apps/web/app/components/hero/hero.css

```
@@ -0,0 +1,348 @@
+}
+
+/* ---------- Status Tag ---------- */
+.status-tag {
+    display: inline-flex;
+    align-items: center;
+    border: 1px solid var(--border-tag);
+    padding: 10px 18px;
+    border-radius: 100px;
+    margin-bottom: 28px;
+
+    opacity: 0;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)